### ISSUE TITLE
where to get finetune's corpors?

### ISSUE THREAD

**Issue by soulbyfeng** [topazape/LSTM_Chem#1]
in your project ,i can t see file to finetune,how to get ? if you have ,cao you give me  a file or tell me how to get it ,thx.

**Comment by topazape** [comment]
I'm sorry to reply late.

When fine-tune turn, you need to put your favorite SMILES set into `./dataset/` directory. (In this context, your SMILES set named `your-favorite-smiles.smi`.)
And edit line like: `finetune_data_filename": "./datasets/your-favorite-smiles.smi"`, in `./configs/LSTMChem_config.json`.
Finally, run `finetune.py`, you get newly generated SMILES in `WantsChems.smi`.

If you don't have relevant SMILES set for fine-tune, please get wet data in your organization.
Or, please remake `./datasets/datasets.smi`, because train dataset chemicals in `datasets.smi` were extracted from **whole** ChEMBL DataBase on condition that chemical shows IC50, EC50, Ki < nM.

For example, if you want to generate new kinase inhibitor, care must be taken you may avoid  kinase inhibitor when you extract from ChEMBL DataBase. And make extracted SMILES set to train dataset and make avoided kinase inhibitor SMILES set for test set (fine-tune set).

**Comment by gwanseum** [comment]
Hello, topazape.

Could you please share the file named 'LSTM_Chem-22-0.42.hdf5'?

Thank you in advance!
